addons/pointwatch/statics.lua

```diff
@@ -81,7 +81,57 @@ other_table = {Crimson=10,Azure=10,Amber=10,Alabaster=10,Obsidian=20}
 dynamis_map = {[185]=city_table,[186]=city_table,[187]=city_table,[188]=city_table,
     [134]=other_table,[135]=other_table,[39]=other_table,[40]=other_table,[41]=other_table,[42]=other_table}
 
-    
+
+job_map = {
```

Comment:
Maybe this and the abbreviation map could be pulled from resources instead of a hard-coded mapping?

Reply:
It should be, this and the `job_abbr_map` should be removed and handled via resource lookups. Should also use `.name` to get language-specific values.